Makefile

```diff
@@ -51,6 +51,17 @@ man:
 shell: build-container
 	$(DOCKER_RUN_DOCKER) bash
 
+test-style: test-fmt test-vet test-lint
+
+test-fmt:
+	test -z "$$(gofmt -s -l . | grep -v ^vendor/ | tee /dev/stderr)"
```

Comment:
Honestly the idea here is the same as the docker upstream repo which in more than a year worked flawlessly for me (apart from the first not coached run). My vim ide is already configured to run at least lint and this is per developer so it's not up for this repository. 
If it's better for you at least, we can add those local makefile targets.